.editorconfig

```diff
@@ -165,3 +165,6 @@ csharp_new_line_before_members_in_object_initializers = true
 csharp_new_line_before_members_in_anonymous_types = true
 
 dotnet_style_require_accessibility_modifiers = always
+
+# Public API
+dotnet_diagnostic.RS0016.severity = warning
```

Comment:
Ah, I read that these now have to be warnings in the .editorconfig to be considered. Just could not connect the dots :)